encoder_odom_publisher:
  ros__parameters:
    use_sim_time: true

    # Joint names from /joint_states
    left_wheel_joint: "joint_wheel_left"
    right_wheel_joint: "joint_wheel_right"

    joint_states_topic: "/joint_states"

    # Frames
    odom_frame: "odom"
    base_frame: "chassis_link"

    # Robot Geometry (meters) - locked calibrated values
    wheel_radius: 0.14
    wheel_separation: 0.4132

    publish_tf: true
    odom_topic: "/odom"
